plugins/downloader/menu.js

```diff
@@ -1,55 +1,24 @@
-const { existsSync, mkdirSync } = require("fs");
-const { join } = require("path");
+const { dialog } = require("electron");
 
-const { dialog, ipcMain } = require("electron");
-const is = require("electron-is");
-const ytpl = require("ytpl");
-const chokidar = require('chokidar');
-const filenamify = require('filenamify');
-
-const { setMenuOptions } = require("../../config/plugins");
-const { sendError } = require("./back");
-const { defaultMenuDownloadLabel, getFolder, presets, setBadge } = require("./utils");
-
-let downloadLabel = defaultMenuDownloadLabel;
-let playingUrl = undefined;
-let callbackIsRegistered = false;
-
-// Playlist radio modifier needs to be cut from playlist ID
-const INVALID_PLAYLIST_MODIFIER = 'RDAMPL';
-
-const getPlaylistID = aURL => {
-	const result = aURL?.searchParams.get("list") || aURL?.searchParams.get("playlist");
-	if (result?.startsWith(INVALID_PLAYLIST_MODIFIER)) {
-		return result.slice(6)
-	}
-	return result;
-};
-
-module.exports = (win, options) => {
-	if (!callbackIsRegistered) {
-		ipcMain.on("video-src-changed", async (_, data) => {
-			playingUrl = JSON.parse(data)?.microformat?.microformatDataRenderer?.urlCanonical;
-		});
-		ipcMain.on("download-playlist-request", async (_event, url) => downloadPlaylist(url, win, options));
-		callbackIsRegistered = true;
-	}
+const { downloadPlaylist } = require("./back");
```

Comment:
This function is not exported, which yields an error when clicking on the menu item

Reply:
Haha sorry I completely forgot that it was an option 😂
Good thing you caught that ! 

I only tested using the youtube popup menu (which is way better since it allows you to actually choose what to download)